pages/privacy.js

```diff
@@ -1,12 +1,21 @@
-import MarkdownContent from '../components/MarkdownContent'
+import MarkdownContent, { getContentProps } from '../components/MarkdownContent'
 
-import privacy from '../public/privacy.md'
+/**
+ * getStaticProps (async) function for statically embedding markdown.
+ * (see https://nextjs.org/docs/basic-features/data-fetching#getstaticprops-static-generation).
+ */
+export function getStaticProps () {
+  return getContentProps(process.env.PRIVACY_POLICY_URL)
+}
```

Comment:
Hrm, I still think it would be better to just include some async/await behavior in this method, rather than feeling like we need to document it in the jsdoc. For example:
```javascript
export async function getStaticProps () {
  const markdown = await getMarkdownContentFromUrl(process.env.PRIVACY_POLICY_URL)
  return {
    props: { markdown }
  }
}
```

Reply:
Updated in https://github.com/ibi-group/otp-admin-ui/pull/36/commits/5be487b20f162352b013e048ab64c4675d4e9e3b.